tuplex/core/include/TraceVisitor.h

```diff
@@ -60,7 +61,12 @@ namespace tuplex {
          // store tracebacks for clean display
          // TODO: could avoid storing duplicates so index by type & line number & column number, yet this not supported yet.
 
+        // was a break statement executed in the ongoing loop?
+        std::vector<bool> _loopBreakStack;
```

Comment:
Do we have test cases for break/continue with all of this unrolling?

Reply:
Added tests.